website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+
+   1. First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`.
+   1. Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`.
+   1. Email Address: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`.
+
+### OneLogin - via PactFlow Connector
+
+1.  On the OneLogin portal page, choose Administration.
+2.  From the Administration page, choose Applications, and then choose Add App.
+
+![Find PactFlow Connector](/saml/saml-onelogin-find-pactflow.png)
+
+3.  In the search bar under Find Applications, enter "pactflow", and then choose the PactFlow Connector
+4.  (Optional) Do any of the following:
+    For Display Name, enter a name and description. For example, PactFlow.
+    For the Rectangular Icon and Square Icon, you can add PactFlow icons.
+    For Description, enter a short summary description.
+5.  Choose Save.
+6.  On the homepage of the new application, choose "More Actions > SAML Metadata" from the main menu
+
+![Configure Onelogin metadata](/saml/saml-onelogin-metadata.png)
+
+### OneLogin - manual setup
+
+#### 1. Create a OneLogin application
+
+1.  On the OneLogin portal page, choose Administration.
+2.  From the Administration page, choose Applications, and then choose Add App.
+3.  In the search bar under Find Applications, enter saml, and then choose SAML Test Connector (Advanced) to open the Add SAML Test Connector page.
+4.  (Optional) Do any of the following:
+    For Display Name, enter a name and description. For example, PactFlow.
+    For the Rectangular Icon and Square Icon, you can add PactFlow icons.
+    For Description, enter a short summary description.
+5.  Choose Save.
+
+#### 2. Edit your OneLogin application configuration
+
+1.  Choose Configuration.
+2.  On the Configuration page, do the following:
+    For RelayState, leave it blank.
+    For Audience, enter `urn:amazon:cognito:sp:ap-southeast-2_x0L1olP0D`
+    Leave Recipient blank.
+    For ACS (Consumer) URL Validator, enter `https://pact-saas-prod-1.auth.ap-southeast-2.amazoncognito.com/saml2/idpresponse`
+    For ACS (Consumer) URL, enter `https://pact-saas-prod-1.auth.ap-southeast-2.amazoncognito.com/saml2/idpresponse`
+    Leave Single Logout URL blank.
+
+#### 3. Edit your OneLogin application's parameters
+
+1.  Choose Parameters.
+    Note: One parameter (NameID (fka Email)) is already listed—this is expected.
+2.  Choose Add parameter to create a new, custom parameter.
+3.  In the New Field dialog, for Field name, enter `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier`
+4.  For Flags, select the Include in SAML assertion check box.
+5.  Choose Save.
+6.  For Value, choose Email from the list.
+7.  Choose Save.
+
+You then need to repeat the steps for the first name and last name.
+
+8.  Choose Add parameter to create a new, custom parameter.
+9.  In the New Field dialog, for Field name, enter First name
```

Comment:
In the New Field dialog, for Field name, enter the First name

Reply:
In this case, I think it should be "First name" verbatim (i.e. the user is instructed to put "First name" into the field, not their actual first name)